kubernetes/rancher/README.md

```diff
@@ -0,0 +1,71 @@
+# Kubernetes with Rancher
+Run Kubernetes with a [Rancher](https://rancher.com/) Docker container.
+
+## Table of Contents
+  - [Description](#description)
+  - [Description](#prerequisites)
+  - [Running with xOpera](#running-with-xopera)
+
+## Prerequisites
+Minimum requirements for running Kubernetes with Rancher are:
+
+* Operating Systems
+  * Ubuntu 16.04 (64-bit)
+  * Red Hat Enterprise Linux 7.5 (64-bit)
+  * RancherOS 1.4 (64-bit)
+* Hardware
+  * 4 GB of Memory
+* Software
+  * Docker v1.12.6, 1.13.1, 17.03.2
+
+To install Docker on your machine you can use our [Docker xOpera example](../docker).
+
+## Description
+The main functionality of this solution is to run a Kubernetes service in a Rancher Docker container on a target 
+machine. The deployed Kubernetes dashboard is the available on `localhost:80` and `localhost:443`. Additionally one 
+[hello-world](https://hub.docker.com/_/hello-world) Docker container is deployed in the Rancher network and it will 
+show up in the dashboard.
+
+There are 2 main Ansible playbooks separated into folders. Usage is explained in the table below:
+
+|    Role      |   Purpose    |
+|:-------------|:-------------|
+| **rancher** | Installs Kubernetes by running Rancher Docker container |
+| **hello-world** | Runs hello-world |
+
+You can modify values of variables stored in `inputs.yaml` to set the suitable parameters. The file with inputs is 
+later passed to xOpera at the beginning of the orchestration. The inputs are explained in a table below.
+
+| Input | Description | Example
+|:-------------|:-------------|:-------------|
+| `host_ip` | IP address of your target (virtual) machine | localhost |
+
+### Running with xOpera
+When running Ansible playbooks with xOpera on a VM you should specify the username that will be used to connect to
+the virtual machine by ssh. This can be done by setting the environment variable `OPERA_SSH_USER`.
+You can also skip `OPERA_SSH_USER` when running xOpera by exporting this variable using: `export OPERA_SSH_USER=root`.
```

Comment:
Is this relevant to the example?

Reply:
`OPERA_SSH_USER` can be used to deploy for instance on a VM but I can remove that last sentence about exporting it because it's logical.

Reply:
No no, that wasn't what I meant. You don't create a VM anywhere, right? So why would you set a user for it, since everything runs on the local machine?

Reply:
This is also true, well for this simple example I will skip this part. All I meant was if user had his own VM he could deploy this example onto this VM.

Reply:
Removed.